### PR TITLE
Add `id` to docs

### PR DESCRIPTION
**Pull request by milwad-dev**

### CODE REVIEW

File: src/Html/Attributes.php [hunk 1]
```diff
@@ -4,17 +4,38 @@
 
 class Attributes
 {
+    /**
```
Comment (sebastiandedeyne) [inline src/Html/Attributes.php:7]: Can you revert these?

File: docs/introduction.md [hunk 2]
```diff
@@ -54,6 +54,26 @@ Menu::new()
 </div
 ```
 
+## Adding id to elements
```
Comment (sebastiandedeyne) [inline docs/introduction.md:57]:
```suggestion
## Adding an ID to elements
```

File: docs/controlling-the-html-output/item-attributes.md [hunk 3]
```diff
@@ -30,3 +32,18 @@ Link::to('#', 'Back to top')
 ```
 
 If you want to add classes or set attributes to all items in a menu, the `Menu` class provides some convenience methods: `addItemClass`, `setItemAttribute`.
+
+## Add id into menu
```
Comment (sebastiandedeyne) [inline docs/controlling-the-html-output/item-attributes.md:36]:
```suggestion
## Adding an ID to a menu
```

File: docs/controlling-the-html-output/item-attributes.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ weight: 1
 
 The `Menu` and `Link` classes use the `HtmlAttributes` trait, which enables you to add attributes to their main elements (respectively the `ul` and `a` tags).
 
+## Add class into menu
```
Comment (sebastiandedeyne) [inline docs/controlling-the-html-output/item-attributes.md:8]:
```suggestion
## Adding classes to a menu
```

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]

File: src/Html/Attributes.php (hunk 1 above)
Comment (milwad-dev) [reply src/Html/Attributes.php]: Oh, This commit for previous PR.
I revert this now.

**Review by sebastiandedeyne — APPROVED** [review]

**Pull request merged**